ci-operator/step-registry/cucushift/baremetal/ipi/cucushift-baremetal-ipi-commands.sh

```diff
@@ -4,10 +4,16 @@ set -o nounset
 set -o errexit
 set -o pipefail
 
+if [ -f "${SHARED_DIR}/kubeconfig" ] ; then
+    export KUBECONFIG=${SHARED_DIR}/kubeconfig
```

Comment:
Why do we have to export KUBECONFIG here?

Reply:
@jhou1  That comes from an conversation with metal-ipi owners, https://coreos.slack.com/archives/CFP6ST0A3/p1634219671275200?thread_ts=1634216796.272000&cid=CFP6ST0A3